src/instruments/src/Common/notification.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable camelcase */
```

Comment:
**suggestion:** move those files to `src/shared/`, as they might be used by non-instrument components such as the FMGC.